tasks/compile.rb

```diff
@@ -0,0 +1,36 @@
+require "compile_to_file"
+
+desc "Compile the application to a single file for deployment"
+task :compile do
+  CompileToFile::App.new do |app|
+
+    app.skip_requires_for("automate_liveness_agent")
+
+    app.add_lib_files(%w{
```

Comment:
We need to copy the files in the same order we'd require them. That _could_ be done by grepping the requires, building a graph, and then topo sorting, but I'm kinda YAGNI on that at the moment.